### DISCUSSION TITLE
Play Video from URL

### DISCUSSION THREAD

**Discussion by naufalkharits** [mpvnet-player/mpv.net#530] [Q&A]
I'm curious, what makes mpv.net can grab a URL?
I tried on mpv, drag and drop a URL but got 🚫 which means unable to grab the URL.

the URL is not from youtube, I dragged it from goindex.

**Comment by systemoutprintlnhelloworld** [comment]
MPV  can did it either,but the [tools ](https://support.qq.com/products/104929/faqs/142988)i used is for chinese ,maybe it is hard for you  to reference ,this link told us how to link this app ,that can play a video and its bullet chat from web ,to MPV.
